database/sql/V059__add_block_number_order_execution.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE order_execution
+    ADD COLUMN block_number bigint NOT NULL;
```

Comment:
IIRC you have to:
1. add the new column with some default value (existing columns don't have a value yet)
2. populate existing rows
3. drop the default modifier

Otherwise postgres will probably complain when you try to add the new column. Or did you try to run the migration locally while you already had some order_execution rows in the DB?

Reply:
I ran a migration locally and it did not complain, however it was done on an empty db.
I adjusted the migration to what you suggest and also did an example run with non-empty db and it works as expected, so I believe this version is the way to go.